### PR TITLE
add Behavior driven tests

### PR DESCRIPTION
**Pull request by AvshRLev**

### CODE REVIEW

**Comment by buzzdan** [comment]
First commit name is not good
Please change

File: __test__/button.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+afterEach(async() => {
+  await browser.close();
+})
+
+  test("test space invaders url", async() => {
+  const url = await page.url();
+  assert(url==="http://localhost:8000/space_invaders");
+});
+
+test("press level 1 button gets level 1", async() => {
```
Comment (buzzdan) [inline __test__/button.test.js:23]: You are actually testing that clicking kevel 1 change the page title

File: package.json [hunk 2]
```diff
@@ -20,11 +20,13 @@
   "homepage": "https://github.com/AvshRLev/gamesproject#readme",
   "dependencies": {
     "-": "0.0.1",
-    "save-dev": "0.0.1-security",
-    "webpack": "^5.36.0",
-    "webpack-cli": "^4.6.0"
+    "save-dev": "0.0.1-security"
```
Comment (buzzdan) [inline package.json:23]: What is this package?

Comment (AvshRLev) [reply package.json:23]: i dont know

File: package.json [hunk 3]
```diff
@@ -19,12 +19,13 @@
   },
   "homepage": "https://github.com/AvshRLev/gamesproject#readme",
   "dependencies": {
-    "-": "0.0.1",
-    "save-dev": "0.0.1-security",
-    "webpack": "^5.36.0",
-    "webpack-cli": "^4.6.0"
+    "-": "0.0.1"
```
Comment (buzzdan) [inline package.json:22]: Mmm 
Can you delete this line?

**Pull request merged**